generateCSV.js

```diff
@@ -56,42 +52,35 @@ const generateData = async (maxFileEntries, numberOfArtists, albumsPerArtist, so
   const numberOfAlbums = numberOfArtists * albumsPerArtist;
   const numberOfSongs = numberOfAlbums * songsPerAlbum;
 
-  let artistCounter = 1;
-  let albumCounter = 1;
-  let songCounter = 1;
-
-  let artistRecords = 0;
-  let albumRecords = 0;
-  let songRecords = 0;
+  let artistCounter = 0;
+  let albumCounter = 0;
+  let songCounter = 0;
 
-  let artistCSV = fs.createWriteStream('./fakeData/artists/artistCSV1.csv');
-  let albumCSV = fs.createWriteStream('./fakeData/album/albumCSV1.csv');
-  let songCSV = fs.createWriteStream('./fakeData/songs/songCSV1.csv');
+  let artistCSV;
+  let albumCSV;
+  let songCSV;
 
-  while (artistRecords < numberOfArtists) {
-    await writeArtistCSV(artistCSV, maxFileEntries);
-    artistRecords += maxFileEntries;
+  for (let i = 0; i < numberOfArtists / maxFileEntries; i += 1) {
+    artistCounter += 1;
     artistCSV = fs.createWriteStream(`./fakeData/artists/artistCSV${artistCounter}.csv`);
+    await writeArtistCSV(artistCSV, maxFileEntries);
     console.log(
       `artist csv ${artistCounter} of ${Math.ceil(numberOfArtists / maxFileEntries)} done`,
```

Comment:
Purely stylistic: All your other console logs are written in one line except for this one